.github/workflows/handler.publish-release-packages.yml

```diff
@@ -42,7 +42,11 @@ jobs:
       - name: Build package
         run: |
           yarn install --frozen-lockfile
+          yarn --cwd packages/sdk-core set-default-subgraph-release-tag
```

Comment:
This solution is not optimal because tests will not be run after this step. It will make the build fail though (i.e. it would have prevented the case we had).

Reply:
As a clarification, the code generation would break.